usecases/scheduled_execution/run_scheduled_execution.go

```diff
@@ -110,54 +121,6 @@ func NewRunScheduledExecution(
 	}
 }
 
-func (usecase *RunScheduledExecution) ScheduleScenarioIfDue(ctx context.Context, organizationId string, scenarioId string) error {
```

Comment:
Moved to another file along with other methods used in it, no other changes (see schedule_scenarios.go)